src/common/options.cc

```diff
@@ -1735,6 +1735,11 @@ std::vector<Option> get_global_options() {
     .add_see_also("osd_pool_default_size")
     .add_see_also("osd_pool_default_min_size"),
 
+    Option("mon_allow_pool_size_one", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(true)//false)
```

Comment:
I think we want to keep the default false

Reply:
ack.